CONTRIBUTING.md

```diff
@@ -90,27 +103,24 @@ in that document may be rejected. If you are unsure if your feature is a
 good fit for virt-manager, please ask on the mailing list before you start
 coding!
 
-
 ## Introductory tasks
 
 Extending the virt-install or virt-xml command line is a good introductory
 task for virt-manager. See [the wiki](https://github.com/virt-manager/virt-manager/wiki)
 for both a patch tutorial, and a list of libvirt `<domain>` XML options
 that still need to be added to our command line.
 
-
 ## Translations
 
 Translations are handled through the Weblate instance hosted by the Fedora Project.
 
-* https://translate.fedoraproject.org/projects/virt-manager/virt-manager/
-* More info about translating as part of Fedora: https://fedoraproject.org/wiki/L10N/Translate_on_Weblate
+* <https://translate.fedoraproject.org/projects/virt-manager/virt-manager/>
+* More info about translating as part of Fedora: <https://fedoraproject.org/wiki/L10N/Translate_on_Weblate>
```

Comment:
Unrelated change, should be done in different commit and explaining what it fixes.